### PR TITLE
Add person

### PR DESCRIPTION
**Pull request by ghost**
[DEMO LINK](https://juajum.github.io/react_person/)

### CODE REVIEW

**Review by Bielichenko — APPROVED** [review]
GJ

File: src/App.js [hunk 1]
```diff
@@ -28,22 +28,9 @@ const alex = {
...
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person {...misha} />
+    <Person {...olya} />
+    <Person {...alex} />
```
Comment (illiarerun) [inline src/App.js:33]: Please, read this:
http://mateacademy.slack.com/archives/C03C2C13KRA/p1657005525754959

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,32 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
```
Comment (illiarerun) [inline src/components/Person/Person.jsx:2]: you haven't commited `Person.scss`, please commit it

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,32 @@
...
+      {age ? (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      ) : null}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`My ${partner} name is ${partnerName}`}
+        </p>
+      ) : 'I am not married'}
```
Comment (illiarerun) [inline src/components/Person/Person.jsx:29]: `I am not married text should be inside `p` tag, too, move your condition inside `<p className="Person__partner">` tag

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,32 @@
...
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age ? (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      ) : null}
```
Comment (illiarerun) [inline src/components/Person/Person.jsx:24]: you can just use 
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by mgrinko — APPROVED** [review]

**Pull request closed without merging**